app/view_objects/results_view.rb

```diff
@@ -67,6 +67,43 @@ def number_of_extra_subjects
     number_of_subjects_selected - NUMBER_OF_SUBJECTS_DISPLAYED
   end
 
+  def location
+    query_parameters["loc"] || "Across England"
```

Comment:
I this it's l equals 2

https://www.find-postgraduate-teacher-training.service.gov.uk/results?l=2&qualifications=QtsOnly,PgdePgceWithQts,Other&fulltime=False&parttime=False&hasvacancies=True&senCourses=False

Reply:
Interesting, cheers, will look closer

Reply:
Also found https://trello.com/c/hh2JmNqP/2950-rails-find-app-doesnt-expand-location-filter-selection-when-changing-query as a result of looking more closely